### ISSUE TITLE
ReactGridLayout requires key id for layout array

### ISSUE THREAD

**Issue by ro-savage** [STRML/react-grid-layout#187]
Maybe I just missed this in the docos, but I wasn't able to find it.

When I created my layout array, I was required to pass a property for **_i_**, but the only mention I could find in the docos was for when using custom keys.

**WORKS**
```JS
import React from 'react'
import ReactGridLayout, { WidthProvider } from 'react-grid-layout'
const ReactGridLayoutWP = WidthProvider(ReactGridLayout)

const LoginPage = () => {
  const layout = [
    { x: 0, y: 0, w: 2, h: 3, i: '1' },
    { x: 2, y: 0, w: 4, h: 3, i: '2', static: true },
    { x: 6, y: 0, w: 2, h: 3, i: '3' },
  ]
  return (
    <ReactGridLayoutWP className="layout" rowHeight={30} layout={layout}>
      <div key={1}>1</div>
      <div key={2}>2 - Static</div>
      <div key={3}>3</div>
    </ReactGridLayoutWP>
  )
}

export default LoginPage
```

**DOESN'T WORK**
```JS
import React from 'react'
import ReactGridLayout, { WidthProvider } from 'react-grid-layout'
const ReactGridLayoutWP = WidthProvider(ReactGridLayout)

const LoginPage = () => {
  const layout = [
    { x: 0, y: 0, w: 2, h: 3 },
    { x: 2, y: 0, w: 4, h: 3, static: true },
    { x: 6, y: 0, w: 2, h: 3 },
  ]
  return (
    <ReactGridLayoutWP className="layout" rowHeight={30} layout={layout}>
      <div key={1}>1</div>
      <div key={2}>2 - Static</div>
      <div key={3}>3</div>
    </ReactGridLayoutWP>
  )
}

export default LoginPage
```

Am I missing something? Or does the documentation need to be updated?
Happy to make a PR to update docs if needed.

**Comment by joayo** [comment]
+1

**Comment by Sunakujira1** [comment]
+1

**Comment by eMerzh** [comment]
on top of that it seems that if the 'i's are not contious (1 then 2 then 3....) the layout will be messed up....
might have a note in the doc as well :)

**Comment by ro-savage** [comment]
This has been closed by PR #191 

Thanks for the feedback :)
